src/BetweenTheBubbles.tbasic

```diff
@@ -32,9 +101,11 @@ function tbMain()
     context.startFrame()
     
     sm.process_screen()
+    
+    TBGL_UseVSync IIF( context.frameRate < maxFrameRate , 0, 1 )
```

Comment:
frameRate-lock loop is not steady. 
Test: without VSync, frameRate 150~200 no drop
with VSync framerate ~60 but, sometimes burst at 150~200, means detecting drop below 60 ? 
see line 95 too.